components/engine_rocks/src/properties.rs

```diff
@@ -535,7 +535,7 @@ pub fn get_range_entries_and_versions(
     cf: &str,
     start: &[u8],
     end: &[u8],
-) -> Option<(u64, u64)> {
+) -> Option<(u64, u64, u64)> {
```

Comment:
I think three fields are enough to justify a formal struct. Now it's easy to make mistake on which one should to use in the caller.

Reply:
Good suggestion.